perf/benchmark/templates/fortio.yaml

```diff
@@ -147,26 +431,46 @@ spec:
         volumeMounts:
         - name: shared-data
           mountPath: /var/lib/fortio
+        - name: nighthawk-test-server-config
+          mountPath: /var/lib/nighthawk 
         image: {{ $.Values.curlImage }}
         args:
         - /bin/sleep
         - infinity
       - name: uncaptured
         securityContext:
           runAsUser: 1
+{{- if eq $.name "fortioclient" }}
         image: {{ $.Values.fortioImage }}
+        volumeMounts:
+        - name: shared-data
+          mountPath: /var/lib/fortio
+        - name: nighthawk-test-server-config
+          mountPath: /var/lib/nighthawk 
         args:
-{{- if eq $.name "fortioclient" }}
         - report
-{{- else }}
-        - server
-{{- end }}
         - -http-port
-        - "8077"
+        - "9076"
```

Comment:
If you change the uncaptured http-port to 9076, you should also change here: https://github.com/istio/tools/blob/master/perf/benchmark/runner/runner.py#L64

Reply:
This is now only used for the fortio reporting service. The nighthawk test server is now listening on the original port. Given that, wdyt, does this look good?

Reply:
oh, i see! if so, it makes sense to me.